packages/form-js-viewer/test/spec/render/components/form-fields/Text.spec.js

```diff
@@ -82,23 +83,27 @@ Some _em_ **strong** [text](#text) \`code\`.
     const formField = container.querySelector('.fjs-form-field');
 
     expect(formField).to.exist;
-    expect(formField.innerHTML).to.eql(`<div class="markup"><div xmlns="http://www.w3.org/1999/xhtml"><h1>h1</h1><h2>h2</h2><h3>h3</h3><h4>h4</h4><h5>h5</h5><h6>h6</h6>
-
-<blockquote>Blockquote</blockquote>
-
-<ul><li>ul li 1</li><li>ul li 2</li></ul>
-
-<ol><li>ol li 1</li><li>ol li 2</li></ol>
-
-<p></p><pre class="code"><code>Some Code</code></pre><p></p>
-
+    expect(formField.innerHTML).to.eql(`<div class="markup"><div xmlns="http://www.w3.org/1999/xhtml"><h1 id="h1">h1</h1>
+<h2 id="h2">h2</h2>
+<h3 id="h3">h3</h3>
+<h4 id="h4">h4</h4>
+<h5 id="h5">h5</h5>
+<h6 id="h6">h6</h6>
```

Comment:
I know the changes are really small due to the transition to `showdown`, but I think we should at least mention there are some changes in the resulting HTML, e.g. via note in
* the commit message
* the changelog 

Just some note on that we change the markdown parser to `showdown` and a quick reference to the specification (if there is some). I think that would help a lot already.

Reply:
Just mentioning it as it seems our users are interested in that (https://camunda-platform.slack.com/archives/C6WGNHV2A/p1678953005263759).

Reply:
Yeah I'll try and break down the commits to make this explicit.